keps/sig-windows/20190714-windows-csi-support.md

```diff
@@ -190,12 +191,12 @@ spec:
             - name: plugin-dir
               mountPath: C:\csi
             - name: csi-proxy-pipe
-              mountPath: \\.\pipe\csi-proxy
+              mountPath: \\.\pipe\csi-proxy-v1alpha1
       volumes:
         - name: csi-proxy-pipe
           hostPath: 
-            path: \\.\pipe\csi-proxy
-            type: null
+            path: \\.\pipe\csi-proxy-v1alpha1
+            type: ""
```

Comment:
I forget if empty string is a valid value.  Do none of the other types fit?

Reply:
empty string is a valid value (that's what missing values default to). there is a `Socket` value, but iiuc this isn't a socket

Reply:
I find it interesting that `type: null` is accepted and works out end2end (as also pointed out in the PR that added named pipe mount support for Windows: https://github.com/kubernetes/kubernetes/pull/69484) but `type: foo` gets rejected with `spec.template.spec.volumes[0].hostPath.type: Unsupported value: "foo": supported values: "", "BlockDevice", "CharDevice", "Directory", "DirectoryOrCreate", "File", "FileOrCreate", "Socket"`. Does the value `null` have special meaning?

Reply:
null is indistinguishable from "not specified", and gets defaulted to ""